PowerEditor/src/Notepad_plus.h

```diff
@@ -387,10 +387,6 @@ friend class FileManager;
                                               // and "Enable session snapshot and periodic backup" is not enabled
                                               // then WM_ENDSESSION is send with wParam == FALSE
                                               // in this case this boolean is set true, so Notepad++ will quit and its current session will be saved 
-
-    bool _isWorkspaceFileLoadedFromCommandLine = false; // Set during Notepad_plus::doOpen if workspace file is opened.
-                                                        // But it is only evaluated during startup, when doOpen
-                                                        // has been called while command line interpretation.
 	ScintillaCtrls _scintillaCtrls4Plugins;

```

Comment:
This removal is interesting too. I finally dared to move the call of the **loadCommandLineParams** function to a later point, where the plug-ins already have been restored. Due to this move the additional effort to switch back to Project Panel 1 after restoring the plug-insis no longer necessary.